lcmtypes/lcmt_osc_tracking_data.lcm

```diff
@@ -0,0 +1,18 @@
+package dairlib;
+
+struct lcmt_osc_tracking_data
+{
+  int32_t y_dim;
+  string name;
+  boolean is_active;
+
+  double y [y_dim];
+  double y_des [y_dim];
+  double error_y [y_dim];
+  double ydot [y_dim];
+  double ydot_des [y_dim];
+  double error_ydot [y_dim];
+  double yddot_des [y_dim];
+  double yddot_command [y_dim];
+  double yddot_command_sol [y_dim];
```

Comment:
I also added this class in #135, so we will have to make sure they are the same once we merge the two PRs

Reply:
Yup. On the same topic, the changes that I made to `OSC` and `OscTrackingData` these two days should be apply to your osc_mbp branch as well. It seems that the differences are 
* new documentation
* renaming `dy` to `ydot` and `ddy` to `yddot`